specification.markdown

```diff
@@ -512,7 +512,7 @@ The version of the EBML Header is found in EBMLVersion. An EBML parser can read
 
 The version of the EBML Body is found in EBMLDocTypeVersion. A parser for the particular DocType format can read the EBML Document if it can read either the EBMLDocTypeVersion version of that format or a version equal or higher than the one found in EBMLDocTypeReadVersion.
 
-# Elements semantics
+# Elements semantic
```

Comment:
This feels wrong.  I think that "Element semantics" would be correct.

Reply:
I concur with "Element semantics" being the way to go here.

Reply:
Fixed